.github/workflows/stellar-expert-release.yml

```diff
@@ -0,0 +1,102 @@
+name: Stellar Expert WASM Release
+
+permissions:
+  id-token: write
+  contents: write
+  attestations: write
+
+on:
+  push:
+    tags:
+      - "v*" # triggered whenever a new tag (previxed with "v") is pushed to the repository
+
+jobs:
+  release-contract-stellar-axelar-example:
+    uses: stellar-expert/soroban-build-workflow/.github/workflows/release.yml@main
+    with:
+      release_name: ${{ github.ref_name }}
+      release_description: "Axelar Example Release"
+      package: "stellar-axelar-example"
+    secrets:
+      release_token: ${{ secrets.GITHUB_TOKEN }}
+
+  release-contract-stellar-axelar-gas-service:
```

Comment:
How does this work exactly if the contracts haven't even been deployed yet? It stores the wasm hash -> source code, and shows it later when deployed?

Reply:
can this be done from our deployment scripts, like evm verification, or is this the recommended way. Add a link to the docs at the top for reference